docs/example-scenario/apps/sap-dev-test.md

```diff
@@ -0,0 +1,109 @@
+---
+title: SAP for a dev/test environment
+description: SAP scenario for a dev/test environment
+author: AndrewDibbins
+ms.date: 7/11/18
+---
+
+# SAP for a dev/test workload
+
+This architectural design shows a set of proven practices for running SAP NetWeaver in a Windows or Linux environment on Azure. The database is AnyDB, the SAP term for any supported DBMS (that isn't SAP HANA). This architecture is deployed with specific virtual machine (VM) sizes that can be changed to accommodate your organization's needs.
+
+## Related use cases
+
+Consider this scenario for the following use cases:
+
+* Non-Critical SAP non-productive workloads (Sandbox, Development, Test, Quality Assurance)
+* Non-critical SAP Business One workloads
+
+## Architecture
+
+![Diagram](media/sap-2tier/SAP-Infra-2Tier_finalversion.png)
+
+This scenario covers the provision of a single SAP system database and SAP application Server on a single virtual machine, the data flows through the scenario as follows:
+
+1. Customers from the Presentation Tier use their SAP gui, or other user interfaces (Internet Explorer, Excel or other web application) on premise to access the Azure based SAP system.
+2. Connectivity is provided through the use of the established Express Route. The Express Route is terminated in Azure at the Express Route Gateway. Network traffic routes through the Express Route gateway to the Gateway Subnet and from the gateway subnet to the Application Tier Spoke subnet and via a Network Security Gateway to the SAP application virtual machine.
```

Comment:
Define what "spoke" means and link to [Hub-Spoke network RA](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/hybrid-networking/hub-spoke)

(But see previous comment, do we need hub/spoke?)

Reply:
added a link, since we have the art already as is - I think we probably leave it with hub/spoke for now; we can look at updating it in the next sprint.  @adamboeglin  can you take this down as a bug?